example_project/measures/BuildResidentialModel/tests/test_build_residential_model.rb

```diff
@@ -66,17 +65,28 @@ def test_hpxml_dir
     # in https://github.com/urbanopt/urbanopt-geojson-gem/blob/develop/lib/urbanopt/geojson/schema/building_properties.json, see:
     # - "hpxml_directory"
 
+    test_folder = File.join(@run_path, __method__.to_s)
```

Comment:
Not important, but fwiw I like using `Pathname` instead of `File.join` for paths. Here's an example: https://github.com/urbanopt/urbanopt-example-geojson-project/blob/develop/example_project/mappers/Baseline.rb#L267

I like the similarity to Python's Pathlib. But that's a me problem, not a you problem.

Reply:
I'll update it.